### PR TITLE
add functions to save on localstorage

### PR DESCRIPTION
**Pull request by mcantuunosquare**
Add functionality to save on local storage the selected users on the contact list page.

### CODE REVIEW

File: frontend/src/app/pages/contact-list/contact-list.component.ts [hunk 1]
```diff
@@ -9,35 +9,28 @@ import { UnicornRewardsApiService } from 'src/app/services/unicorn-rewards-api.s
   styleUrls: ['./contact-list.component.css']
 })
 export class ContactListComponent implements OnInit {
+  private selectedUsersFile:string = "./selected_users.json";
```
Comment (jrodrigoav) [inline frontend/src/app/pages/contact-list/contact-list.component.ts:12]: What is this file for?

**Review by jrodrigoav — CHANGES_REQUESTED** [review]
Looks good, though unit tests for the service should include tests for the additional methods that control the list.

File: frontend/src/app/pages/contact-list/contact-list.component.ts (hunk 1 above)
Comment (marcocantugea) [reply frontend/src/app/pages/contact-list/contact-list.component.ts:12]: sorry, I forgot to delete this line.

**Review by jrodrigoav — APPROVED** [review]
Looks good

**Pull request merged**